MibTeX/src/de/mibtex/export/typo3/Filters.java

```diff
@@ -0,0 +1,63 @@
+package de.mibtex.export.typo3;
+
+import java.util.Arrays;
+import java.util.function.Predicate;
+
+public class Filters {
+	public final static String ThomasThuem = "Thomas Th�m";
+	public final static String ChicoSundermann = "Chico Sundermann";
+	public final static String TobiasHess = "Tobias He�";
+	public final static String PaulMBittner = "Paul Maximilan Bittner";
+	
+	public final static Predicate<Typo3Entry> Is_misc = b -> b.type.equals("misc");
```

Comment:
Final variables are supposed to have upper case letters

Reply:
You mean Capslock only? IS_MISC, TOBIASHESS, ...?

Reply:
UPPER_CASE as I teached in my SE lecture :)

Reply:
done